### PR TITLE
Add CI for RNTester

### PR DESCRIPTION
**Pull request by mobily**
This PR adds CI for `RNTester` (issue: #54).

Log:
https://travis-ci.org/reasonml-community/bs-react-native/builds/337167048?utm_source=github_status&utm_medium=notification

### CODE REVIEW

**Comment by mobily** [comment]
@wokalski ready for review :)

File: RNTester/package.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build": "bsb -clean-world -make-world",
+    "build": "node ../node_modules/.bin/bsb -make-world",
+    "clean-build": "node ../node_modules/.bin/bsb -clean-world -make-world",
     "packager:start": "node node_modules/react-native/local-cli/cli.js start",
-    "watch": "bsb -make-world -w",
+    "watch": "node ../node_modules/.bin/bsb -make-world -w",
```
Comment (wokalski) [inline RNTester/package.json]: what is this?

File: RNTester/package.json [hunk 2]
```diff
@@ -15,8 +16,7 @@
   },
   "devDependencies": {
     "babel-jest": "20.0.0",
-    "babel-preset-react-native": "1.9.1",
-    "fs-extra": "^3.0.1"
+    "babel-preset-react-native": "1.9.1"
```
Comment (wokalski) [inline RNTester/package.json]: Why do we need babel transforms at all?

File: RNTester/package.json (hunk 1 above)
Comment (mobily) [reply RNTester/package.json]: ok, in the local environment `bsb` is usually is installed globally (according to this: https://reasonml.github.io/docs/en/quickstart-javascript.html), and here we must use `bsb` installed in the root `node_modules` (`bsb` belongs to the `bs-platform` package)

File: RNTester/package.json (hunk 2 above)
Comment (mobily) [reply RNTester/package.json]: we need this to build the iOS/Android app correctly :)

File: RNTester/package.json (hunk 1 above)
Comment (wokalski) [reply RNTester/package.json]: I get that but do we need to invoke node instead of just the path itself?

Comment (mobily) [reply RNTester/package.json]: 👍 fixed!

**Comment by wokalski** [comment]
Thank you for that. It's been broken for some time now.

**Comment by mobily** [comment]
thanks! 🎉

**Pull request merged**